pilot/test/mock/config.go

```diff
@@ -118,6 +120,27 @@ var (
 			}},
 		}},
 	}
+
+	// ExampleEndUserAuthenticationPolicySpec is an example EndUserAuthenticationPolicySpec
+	ExampleEndUserAuthenticationPolicySpec = &mccpb.EndUserAuthenticationPolicySpec{
+		Jwts: []*mccpb.JWT{
+			{
+				Issuer: "https://issuer.example.com",
+				Audiences: []string{
+					"audience_foo.example.com",
+					"audience_bar.example.com",
+				},
+				JwksUri:                "https://www.example.com/oauth/v1/certs",
+				ForwardJwt:             true,
+				PublicKeyCacheDuration: types.DurationProto(5 * time.Minute),
+				Locations: []*mccpb.JWT_Location{{
+					Scheme: &mccpb.JWT_Location_Header{
+						Header: "Bearer",
```

Comment:
Don't use "Bearer".  It is confusing.   "Bearer" is not a http header name.  It is a prefix in the http header value.

Reply:
I already fixed this elsewhere but missed this spot - thanks.